roles/etcd_client_certificates/meta/main.yml

```diff
@@ -14,3 +14,4 @@ galaxy_info:
   - system
 dependencies:
 - role: etcd_common
+- role: delegated_serial_command
```

Comment:
Putting all of our custom modules into a single role sounds smart to me

```
.
└── openshift-ansible
    └── roles
        └── openshift_modules
            ├── library
            │   ├── delegated_serial_command.py
            │   ├── modify_yaml.py
            │   ├── ...
            │   └── rpm_q.py
            ├── meta
            │   └── main.yml
            └── README.md
```

Each other role can then just set a dependency on the `openshift_modules` role then, right?